xcp_d/utils/doc.py

```diff
@@ -209,6 +209,34 @@
     List of paths to atlas-specific ROI-to-ROI correlation files.
 """
 
+docdict["process_surfaces"] = """
+process_surfaces : :obj:`bool`, optional
+    If True, a workflow will be run to warp native-space (fsnative) reconstructed cortical
+    surfaces (surf.gii files) produced by Freesurfer into standard (fsLR) space.
+    These surface files are primarily used for visual quality control.
+    By default, this workflow is disabled.
+
+    .. list-table:: The surface files that are generated by the workflow
+        :align: left
+        :header-rows: 1
+        :stub-columns: 1
+
+        * - Filename
+          - Description
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_hcpinflated.surf.gii``
+          - Something
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_hcpmidthickness.surf.gii``
+          - Something
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_hcpveryinflated.surf.gii``
+          - Something
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_midthickness.surf.gii``
+          - Something
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_pial.surf.gii``
+          - Something
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_smoothwm.surf.gii``
+          - Something
```

Comment:
@madisoth could you fill in the gaps on this? I don't know how to describe each, or what they're used for.